WIP - Reworking the grant_revoke_gem_authority script (#47)
samvera/maintenance: data/definitive-rubygems-authors-list.txt

# This file has the definitive and declarative list of Samvera ruby

> # gems owners.  Each line must either be either an email address
> # of the associated Rubygems profile.

Reviewer: I think there may be a typo in this second sentence. How about:

```suggestion
# gems owners.  Each line must either be a rubygems.org email address
# or username.
```

aaron.collier@stanford.edu
adam.wead@protonmail.com
anna3lc@gmail.com
armintor@gmail.com
bess
bmquinn

> cam156@psu.edu

Reviewer: Recommend removing this address as @carolyncole no longer works at Penn State. If she still works w/ Samvera-y things, she can supply her new rubygems.org username/email address, I reckon. (Hi, Carolyn! Wishing you well!)

> chris@cbeer.info

Reviewer: hey @cbeer, do you want keys to the samvera castle or nah

cjcolvar
dan.coughlin@gmail.com
dchandekstark@gmail.com
digger250@gmail.com

> drew@codhicitta.com

Reviewer: Not familiar with this one. Maybe someone else can vouch.

> dweber.consulting@gmail.com

Reviewer: Please remove this one, and revoke his access to our gems.

elrayle
escowles@ticklefish.org
glen.horton@gmail.com

> jenlindner@gmail.com

Reviewer: Are you doing Samvera-y things, @jenlindner? (And, hi!)

> jeremy.n.friesen@gmail.com

Reviewer: I see that `jamielittle@outlook.com` (DCE's @little9) isn't in this list, but perhaps he should be.

> jessie.keck@gmail.com

Reviewer: hey @jkeck, do you want keys to the samvera castle or nah

Reviewer 2: I find it useful to be able to contribute to some gems in samvera land that we depend on from time to time (although that seems to be fewer and father between).  I don't particularly mind, but if you all are trying to clean things up, feel free to 🔪 me.

Reviewer: Nope, I think it's fine for you to retain access. Thought it prudent to ask first! Thanks, Jessie.

johnson.tom@gmail.com

> jonathan@dnil.net

Reviewer: hey @jrochkind , do you want keys to the samvera castle or nah

Reviewer 3: Thanks! Same answer as jkeck! I find it useful to contribute to some gems in samvera we depend on from time to time, but if you are trying to clean things up feel free to get rid of me! (from rubygem owners; I guess from samvera github commit auth is another thing?)

Reviewer: @jrochkind Same response from me, then: Nope, I think it's fine for you to retain access. Thought it prudent to ask first! Thanks, J-Rock. :)

> jpstroop@gmail.com

Reviewer: I suspect Stroop's ruby gem days are behind him.

jrgriffiniii@gmail.com

> lawhorkl@mail.uc.edu

Reviewer: Does Kyle still do Samvera things, @hortongn @scherztc ?

leftwing@alumni.rutgers.edu
mark@curationexperts.com
mbklein@gmail.com
mkorcy@gmail.com

> ndushay@stanford.edu

Reviewer: hey @ndushay , do you want keys to the samvera castle or nah

> ohiocore@gmail.com

Reviewer: Joe no longer works in the Samvera community, so you can remove this address.

rachelkg@gmail.com
randalldfloyd@gmail.com

> revgum@gmail.com

Reviewer: Josh G. no longer works in the Samvera community, so you can remove this address.
